RedacteurPortaal.Tests/DtoBuilder.cs

```diff
@@ -10,6 +10,10 @@ namespace RedacteurPortaal.Tests;
 
 public static class DtoBuilder
 {
+    static DtoBuilder()
+    {
```

Comment:
Waarvoor deze ctor? dit is een static class

Reply:
Rider heeft dit gedaan 💯